packages/dappmanager/src/modules/https-portal/index.ts

```diff
@@ -67,6 +68,16 @@ export class HttpsPortal {
 
     // Edit compose to persist the setting
     addNetworkAliasCompose(container, externalNetworkName, aliases);
+
+
+    // Check whether DNP_HTTPS compose has external network persisted
+    const httpsComposePath = ComposeEditor.getComposePath(params.HTTPS_PORTAL_DNPNAME, true)
+    const editor = new ComposeEditor(ComposeEditor.readFrom(httpsComposePath))
+
+    if(editor.getComposeNetwork(externalNetworkName) === null) {
+      const httpsExternalAlias = getExternalNetworkAlias(httpsPortalContainer)
+      addNetworkAliasCompose(httpsPortalContainer, externalNetworkName, [httpsExternalAlias])
```

Comment:
The compose has to be `docker-compose up` to take effect

Reply:
To take effect as in change compose file or as in connect to the network? If it is the latter we already take care of that few lines up.

Reply:
Oh right, should be okay then